rmf_task/include/rmf_task/Task.hpp

```diff
@@ -84,16 +85,52 @@ class Task::Booking
     ConstPriorityPtr priority_,
```

Comment:
Unrelated but probably a good time to remove the trailing `_`

Reply:
https://github.com/open-rmf/rmf_task/pull/81/commits/bbb3689c63cbc89d5f17b299f5ba87071dc75164